JBPM-5996: Click is not ignored in actions column on list views

rbarriuso opened the pull request.

ederign commented:
ok to test

cristianonicolai commented:
jenkins please retest this.

cristianonicolai commented on jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/list/AbstractMultiGridView.java:
```diff
@@ -399,5 +400,20 @@ public int getRefreshValue() {
+    }
+    
+    public boolean isSelectionIgnoreColumn(int colIx){
+        boolean ret = false;
```
I would suggest using a stream like alternative: .stream.filter.findFirst.isPresent or similar.

rbarriuso replied:
Good idea

rbarriuso commented:
I've rebased, added tests and moved the "SelectionIgnoreColumn" functionality to the `ExtendedPagedTable` class, as I realized that it fits better there than in `AbstractMultiGridView`.

rbarriuso commented:
Applied functionality to `ProcessDefinitionListViewImpl`
